parm/config/gefs/config.base

```diff
@@ -103,6 +103,7 @@ export ROTDIR="@COMROOT@/${PSLOT}"
 
 export ARCDIR="${NOSCRUB}/archive/${PSLOT}"
 export ATARDIR="@ATARDIR@"
+export FETCHDIR="@FETCHDIR@"  # DG adding 121724
```

Comment:
These comments are fine while testing, but they shouldn't be in the final PR.  Instead, a comment on what the `FETCHDIR` is for would be useful.

```suggestion
export FETCHDIR="@FETCHDIR@"  # HPSS or local directory where IC tarball(s) can be found.
```

Reply:
I agree (done)